components/model_analyzer/TorchBenchAnalyzer.py

```diff
@@ -0,0 +1,120 @@
+
+from components.model_analyzer.dcgm.dcgm_monitor import DCGMMonitor
+from components.model_analyzer.tb_dcgm_types.da_exceptions import TorchBenchAnalyzerException
```

Comment:
Why this directory is called `tb_dcgm_types`? If they are from Nvidia, let's keep its original name